apps/application-status/pages/sprints/[team].vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div>
+    <ul>
+      <li v-for="team in curTeams" :key="team.id">
+        <NuxtLink :to="team.path">
+          {{ team.title }}
+        </NuxtLink>
+      </li>
+    </ul>
+    <button @click="changeEndCursor">
+      Next
+    </button>
+    <button @click="changeStartCursor">
+      Prev
+    </button>
+    <p> {{ teamName }} </p>
+    <pre>
+            {{ display }}
+        </pre>
+  </div>
+</template>
+
+<script lang="ts">
+import getSprints from '../../helper/getSprints'
+import teams from '../../data/TeamsData'
+import { PageInfo, TeamSprints } from '../../interface/interfaces'
+import { getReleases } from '../../composables/getReleases'
+export default {
+  data () {
+    const route = useRoute()
+    const teamName = route.params.team
+    const display: TeamSprints = {
+      totalCount: 0,
+      pageInfo: {
+        endCursor: '',
+        hasNextPage: false,
+        hasPreviousPage: false,
+        startCursor: ''
+      },
+      sprints: []
+    }
+    const pageInfo: PageInfo = {
+      startCursor: '',
+      endCursor: '',
+      hasNextPage: false,
+      hasPreviousPage: false
+    }
+    const prev: string[] = ['']
+    const curTeams = teams
+    const boardId = ''
+    const keyWord = ''
+    return {
+      teamName,
+      pageInfo,
+      display,
+      prev,
+      curTeams,
+      boardId,
+      keyWord
+    }
+  },
+  beforeMount () {
+    this.displayData()
+  },
+
+  methods: {
+    async displayData () {
+      if (this.boardId === '' && this.keyWord === '') {
+        if (this.teamName === 'entity') {
```

Comment:
can we not hardcode team names here and use enum values instead

Reply:
yes, I believe so, I'll try